src/lib/logic/stores/config.ts

```diff
@@ -38,7 +36,28 @@ export function setFormStyles(config: FormStyles = {}) {
   setContext("formStyles", styles);
 }
 
-export function setConfig({ fields, form }: Config) {
+export function setStylesVariables(config: Config["vars"] = {}) {
+  injectGlobal`
+    :root {
+      --s-form-primary: ${config.primary ?? "#CCCCCC"};
+      --s-form-secondary: ${config.secondary ?? "#FFFFFF"};
+      --s-form-accent: ${config.accent ?? "#000000"};
+      --s-form-error: ${config.error ?? "#FF0000"};
+      --s-form-warning: ${config.warning ?? "#FFA500"};
+      --s-form-success: ${config.success ?? "#008000"};
+      --s-form-shadow: ${config.shadow ?? "0 0 5px 0 rgba(0, 0, 0, 0.5)"};
+      --s-form-border: ${config.border ?? "1px solid var(--s-form-primary)"};
+      --s-form-text: ${config.text ?? "#000000"};
+      --s-form-placeholder: ${config.placeholder ?? "#B8B2B2"};
+      --s-form-text-error: ${config.texterror ?? "#FFFFFF"};
+      --s-form-radius: ${config.radius ?? "4px"};
+      --s-form-font: ${config.font ?? "system-ui, sans-serif"};
+    }
+  `;
```

Comment:
🚫 **[eslint]** <[@typescript-eslint/no-unused-expressions](https://typescript-eslint.io/rules/no-unused-expressions)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Expected an assignment or function call and instead saw an expression.